code/index.html

```diff
@@ -1,22 +1,96 @@
 <!DOCTYPE html>
 <html>
-  <head>
-    <meta charset="utf-8" />
-    <title>Project 2: The News Site</title>
-    <link rel="stylesheet" href="./style.css" />
-    <!-- dont forget the meta tag for responsiveness -->
-  </head>
-  <body>
-    <header>
-      <!-- Some kind of logo and navbar -->
-    </header>
 
-    <section class="big-news">
-      <!-- The big news content -->
-    </section>
+<head>
+  <meta charset="utf-8" />
+  <title>Project 2: The News Site</title>
+  <link rel="stylesheet" href="./style.css" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+</head>
 
-    <section class="">
-      <!-- The other news  -->
-    </section>
-  </body>
-</html>
+<body>
+  <header>
+    <nav>
+      <img class="logo" src="./icons256.png" alt="logo">
+      <h1 class="site-name">NEWS TODAY</h1>
+      <div class="navigation" id="navigation-bar">
+        <ul>
+          <li><a href="default.asp">Log In</a></li>
+          <li><a href="subscribe.asp">Subscribe</a></li>
+          <li><a href="contact.asp">Contact</a></li>
+        </ul>
+    </nav>
+  </header>
+  <hr>
```

Comment:
What is  `<hr>`  A tag for  making space? Ah, read about it now. Good to learn Thank you!